central-pet-front/.env.test

```
@@ -1 +1,3 @@
 VITE_AUTH_STRATEGY=session
+VITE_SITE_NAME=Centrau Pet
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)